src/nunit.analyzers/nunit.analyzers.csproj

```diff
@@ -5,7 +5,10 @@
     <TargetFramework>netstandard1.6</TargetFramework>
     <PackageTargetFallback>portable-net45+win8</PackageTargetFallback>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
I guess I should remove this. For some reason there were a lot of "errors" in the test case generated by SonarLint, which would be warnings otherwise. So I was looking for a way to turn it off.